src/general/WorldGenerationSettings.cs

```diff
@@ -124,4 +134,48 @@ public override string ToString()
             $", Easter eggs: {EasterEggs}" +
             "]";
     }
+
+    /// <summary>
+    ///   Generates a formatted string containing translated difficulty details.
+    /// </summary>
+    public string GetTranslatedDifficultyString()
+    {
+        string translatedDifficulty = Difficulty is DifficultyPreset difficulty ?
+            difficulty.Name :
+            TranslationServer.Translate("DIFFICULTY_PRESET_CUSTOM");
+
+        return string.Format(CultureInfo.CurrentCulture, TranslationServer.Translate("DIFFICULTY_DETAILS_STRING"),
+            translatedDifficulty,
+            MPMultiplier,
+            AIMutationMultiplier,
+            CompoundDensity,
+            PlayerDeathPopulationPenalty,
+            TranslationServer.Translate("PERCENTAGE_VALUE").FormatSafe(Math.Round(GlucoseDecay * 100, 1)),
+            OsmoregulationMultiplier,
+            TranslationHelper.TranslateBool(FreeGlucoseCloud),
+            TranslationHelper.TranslateBool(PassiveGainOfReproductionCompounds),
+            TranslationHelper.TranslateBool(LimitReproductionCompoundUseSpeed));
```

Comment:
I see now where the TranslateBool is used, so I'd say rename this to "TranslateFeatureFlag" and then the keys would be "FEATURE_ENABLED" and "FEATURE_DISABLED" and maybe even the English text could be, "enabled" and "disabled" to make the output more natural sounding to non-programmers.